### DISCUSSION TITLE
Mass Loss rate

### DISCUSSION THREAD

**Discussion by PurnaPushkalaSomasundaram** [firemodels/fds#12743] [Q&A]
Hello! This Is Purna Pushkala from Politecnico di Milano. I am using FDS for my thesis work. I have made a model of a corn oil tank of 0.8m in diameter and 0.25 m in height and have given inputs for the pool fire studies. The mass loss rate is obtained from the experimental studies as 0.039028 kg/(m²·s). It is found that the mass loss is double times what the actual mass loss is supposed to be. I don't know if this is right or wrong. I need help with this issue.

<img width="1093" alt="mrl" src="https://github.com/firemodels/fds/assets/166021559/9d4192f1-4318-494c-b7e3-241beeda2605">

**Comment by mcgratta** [comment]
What assumptions have you made in your model? Soot yield? Absorption coefficient? Liquid properties? Chemical composition of the evaporated vapors?

**Reply by PurnaPushkalaSomasundaram** [reply]
I can give you the input. Soot yield= 0.038
CO Yield=0.02
HCN yield = 0.01
heat of combustion=40630 kJ/kg
Mass loss rate=0.039028 kg/m2s
Corn oil = C22H25O2N

**Comment by drjfloyd** [comment]
We have no idea what your inputs are, how are we supposed to help you? Make sure you have read the FDS documentation and looked at any relevant example files. If that does not help, attach a simple input file that demonstrates the problem.

**Reply by PurnaPushkalaSomasundaram** [reply]
I can attach a copy of the input file FDS of my model. Hope you can help me with that.
[2_git.txt](https://github.com/firemodels/fds/files/14872638/2_git.txt)

**Reply by drjfloyd** [reply]
This is not the FDS input file.

**Reply by PurnaPushkalaSomasundaram** [reply]
Sorry, I think the file is not the correct one. I will send you the correct one.
[2.odt](https://github.com/firemodels/fds/files/14872718/2.odt)

**Reply by PurnaPushkalaSomasundaram** [reply]
sorry it was in an FDS text file and couldn't upload it so sent in Word doc. with the coding in it

**Reply by rmcdermo** [reply]
@PurnaPushkalaSomasundaram Rename the FDS input file (casename.fds) to (casename.txt) and then it will upload to GitHub.

**Reply by PurnaPushkalaSomasundaram** [reply]
[PurnaPushkalaSomasundaram..txt](https://github.com/firemodels/fds/files/14872850/PurnaPushkalaSomasundaram.txt)

**Comment by mcgratta** [comment]
```
GEOMETRY='CYLINDRICAL',
 LENGTH=0.25,
 RADIUS=0.4/
```
These parameters are not appropriate for your  case. The liquid surface is not a cylinder.

**Reply by PurnaPushkalaSomasundaram** [reply]
I have made rectangular meshes over the dimensions as FDS cannot recognise the circular dimensions. So what am I supposed to write there?

**Reply by PurnaPushkalaSomasundaram** [reply]
Should I use default?

**Reply by rmcdermo** [reply]
Just do this
```
&SURF ID='Fire ',
      COLOR='RED',
      MLRPUA=0.039028,
      RAMP_Q='Fire _RAMP_Q',
      TMP_FRONT=140.0/
```

**Reply by PurnaPushkalaSomasundaram** [reply]
okay.  There is no need to mention the geometry ?

**Reply by rmcdermo** [reply]
Not is this case.  You are creating the geometry with your OBST lines.  The GEOMETRY parameter on SURF is needed when the SURF is applied to a subgrid particle and then FDS needs to know what the geometry is so it can apply the appropriate heat transfer correlations.

In fact, @mcgratta  it would be good if we trapped the use of GEOMETRY for a SURF that is not applied to a particle, if that is not too difficult.

**Reply by PurnaPushkalaSomasundaram** [reply]
Okay, I did apply the same coding as you said. But the graph of mass loss is still the same as I can show you here.
<img width="374" alt="MLR1" src="https://github.com/firemodels/fds/assets/166021559/285f16f6-249b-4dad-99b0-3d823d9f90da">

**Reply by rmcdermo** [reply]
Isn't that following your ramp?

**Reply by rmcdermo** [reply]
Try using HRRPUA=MLRPUA*HOC instead of MLRPUA.  That is the appropriate input for RAMP_Q.

**Reply by PurnaPushkalaSomasundaram** [reply]
the problem is that I have designed the pool fire tank for a 0.4 m radius, which is in terms of area 3.14*0.4*0.4=0.5024 m2. As we multiply with the mass loss rate  .i.e  0.039028 kg/m2s * 0.5024m2 = 0.196 kg/s (mass loss). We need to obtain this value of mass loss from the given area but we can see that it is exceeding this value. What would be the possible reason for this?

**Reply by rmcdermo** [reply]
I don't see where it is exceeding this value.  It is following your ramp.

Please take a step back and start simple.  Create one OBST with the top area of 1 m2.  Apply HRRPUA without a ramp.  Convince yourself that you see the correct mass loss rate (see MLR from _hrr.csv).

Then... add back your ramp.  Check.

Then... create a circular geometry.  Check.

**Reply by PurnaPushkalaSomasundaram** [reply]
okay thank you so much and I will try this and let you know.

**Reply by drjfloyd** [reply]
MRLPUA uses RAMP_Q.

**Reply by PurnaPushkalaSomasundaram** [reply]
okay, so the graph that I have posted in the first question is correct?

**Reply by PurnaPushkalaSomasundaram** [reply]
Hello! I tried a sample as you said. I can show you the FDS file with the MLR graph. 
[model_git.txt](https://github.com/firemodels/fds/files/14881088/model_git.txt)
<img width="371" alt="image" src="https://github.com/firemodels/fds/assets/166021559/7b0ec7ae-0917-4af4-9b35-df76a9686001">
